spec/lib/portus/ldap_spec.rb

```diff
@@ -332,6 +332,15 @@ def load_configuration_test
       ]
     end
 
+    let(:multiple_emails) do
+      [
+        {
+          "dn"     => ["ou=users,dc=example,dc=com"],
+          "email"  => ["user1@example.com","user2@example.com"]
```

Comment:
Rubocop is complaining about not adding space after the comma. So, this should be instead:

```ruby
["user1@example.com", "user2@example.com"]
```